packages/styles/src/elements/paragraph.scss

```
@@ -0,0 +1,9 @@
+@use 'sass:map';
+@use '../functions/tokens';
+@use '../tokens/elements';
+
+tokens.$default-map: elements.$post-paragraph;
+
+p {
+  margin-block-start: tokens.get('post-paragraph', 'margin-block-start');
```

Just for information, you do not need to write the `post-` prefix when using the `tokens.get` function (but it works either way): 
```suggestion
  margin-block-start: tokens.get('paragraph', 'margin-block-start');
```